testing-overhead/src/test/java/io/opentelemetry/util/NamingConvention.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.util;
+
+import io.opentelemetry.agents.Agent;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class NamingConvention {
```

Comment:
Javadoc please

Reply:
For a test utility class?  Ok, I suppose I can do that.....

Reply:
Utility class is irrelevant here. We need to explain what is this naming convention and why do we need it.